.ci/test_r_package_windows.ps1

```diff
@@ -181,6 +185,11 @@ if ($env:COMPILER -ne "MSVC") {
   Get-Content -Path "$INSTALL_LOG_FILE_NAME"
   Write-Output "----- end of build and install logs -----"
   Check-Output $install_succeeded
+  # some errors are not raised above, but can be found in the logs
+  if (Get-Content "$INSTALL_LOG_FILE_NAME" | Select-String -Pattern "ERROR" -Quiet) {
+      echo "ERRORs have been found installing lightgbm"
```

Comment:
Oops! Seems that during VS trials and **errors** (we use them to find any supported VS version) we trigger this condition:
```
Trying 'Visual Studio 16 2019'

-- Configuring incomplete, errors occurred!
```

Reply:
I always forget that `Select-String` is case-insensitive by default 🙃

Reply:
> > LGTM! Looks like we cannot rely on Run-R-Code-Redirect-Stderr...
> 
> Oh wait, I see why! I might be able to fix this by fixing `Run-R-Code-Redirect-Stderr`.
> 
> My next few commits here will be to test...I'll comment when it's ready for another review

ok sadly my experiment failed. I thought I just needed to add a return or Exit in `Run-R-Code-Redirect-Stderr`  but that didn't work. All of the ways we have to catch errors makes it really complicated :/

Reply:
fixed!